### DISCUSSION TITLE
Give the "No Scope" mod an option to toggle between allowing cursor ripples.

### DISCUSSION THREAD

**Discussion by Nyanaro** [ppy/osu#24710] [Ideas]
I'd like to play using the No Scope mod  with cursor ripples enabled. Currently the mod disables cursor ripples. Would be nice if there was a toggle.

**Comment by nagi-desuuu** [comment]
Someone can cheese this by spamming the buttons to reveal the position of the cursor, which defeats the whole purpose of _**NO SCOPE**_.
It's _**NO SCOPE**_ for a reason!

**Reply by Nyanaro** [reply]
So I'll just have to go back to skinning my cursor to be invisible and having an awful time in song select just so I can play without a cursor with ripples enabled, awesome :+1: just like in stable

Not to mention spamming buttons is very impractical and having cursor ripples doesn't in fact defeat the purpose of No Scope. Please say that again once you've actually tried playing.

You can see which toggles are enabled in the results screen btw

**Reply by bdach** [reply]
removing difficulty from a mod via a configuration option, and therefore intentionally creating an imbalanced playing field that makes the mod easier for people who know about the option, just to satisfy your visual preferences, is certainly one of the opinions of all time

like sure you may want the game to look that way but this would be the absolute worst way to go around it

**Reply by Nyanaro** [reply]
The mod already has a slider for when the cursor disappears, which might I add, also reduces the difficulty of the mod (If you consider playing without a cursor as increasing the difficulty).

I use the mod as a quality of life option to have access to a cursor in the menus, while getting to play without a cursor ingame, which is how I like to play.

If you think cursor ripples make the mod easier, that's fine. The toggle would be visible in the scorescreen for all to see, just like the amount of combo you selected before the cursor disappears.

**Reply by cl8n** [reply]
bdach i dont really understand your opinion here. the whole point of settings on mods is to further tune their effects and difficulty. it is obviously easier to play no scope with cursor ripples, and that can be appropriately reflected anywhere that concerns difficulty, like score or pp (if no scope will ever not be identical to nomod for those purposes). beyond that I don't see any issue...

or maybe something has changed in the last year wrt how mod settings are displayed on scores and that's why this comment isn't making sense to me :thinking:

**Reply by Tenexxt** [reply]
I'd make the riple work the way it would be randomly offset with NS so if you even tried to cheese it you'd get the wrong Info but you'd also get a visual for aesthetic purposes.

**Comment by peppy** [comment]
Is this because you're using touch input or something? That's the only valid case I can think of.

**Reply by bdach** [reply]
now that you mention that I'm not sure we even *want* that to be a selectable mod on touch devices because it would basically do nothing, wouldn't it? if touching screen, don't need cursor...

**Reply by Nyanaro** [reply]
I like playing without a cursor, but since I want to have a cursor in the game menus, I normally skin it and replace it with an 8x8 pixel cursor that is completely black, so I can still navigate the menus, albeit with more effort.

I also like playing with cursor ripples enabled, since it provides me useful gameplay feedback. I consider the "No Scope" mod a quality of life feature, as it allows me to truly play without a cursor ingame, while still navigating the menus without issue. All it's lacking is a toggle for cursor ripples.

edit: No I don't play using a touch device, I use tablet and I've been playing the game without a cursor on and off for 6 years at this point.
